doc-content/enterprise-only/openshift/template-deploy-ldap-proc.adoc

```diff
@@ -18,6 +18,11 @@ You must create a user with the username and password configured in the secret f
 .Procedure
 . Set the `AUTH_LDAP*` parameters of the template. These parameters correspond to the settings of the `LdapExtended` Login module of {EAP}. For instructions about using these settings, see https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.0/html-single/login_module_reference/#ldapextended_login_module[LdapExtended login module].
 +
+[NOTE]
```

Comment:
Minor suggestion - Change article of space and remove "character". IBMSG appears to refer to the space character as "space" or "a space". I tried rewriting the note, please disregard if it doesn't convey the information correctly.

s/If you want to enable LDAP failover, you can set two or more LDAP server addresses in the AUTH_LDAP_URL parameter, separated by the space character./To enable LDAP failover, set two or more LDAP server addresses in the AUTH_LDAP_URL parameter with each address separated by a space.